test/generate_property_json.js

```diff
@@ -157,42 +180,60 @@ function init (ops) {
   ops.push({ op: 'mkdir', path: '../outside' })
 }
 
-function start (ops) {
+function startWatcher (ops) {
   running = true
-  ops.push({ op: 'start' })
+  ops.push({ op: 'start_watcher' })
+}
+
+function startClient (ops) {
+  ops.push(sleep())
+  ops.push({ op: 'start_client' })
 }
 
-function run (ops) {
+function run (ops, availableOps) {
   const n = faker.random.number(nbRunOps)
   for (let i = 0; i < n; i++) {
-    const op = freq([
-      [5, createNewDir],
-      [3, createNewFile],
-      [1, recreateDeletedDir],
-      [1, recreateDeletedFile],
-      [1, updateFile],
-      [2, mvToNewPath],
-      [3, mvToDeletedPath],
-      [1, mvToOutside],
-      [1, mvFromOutside],
-      [5, rm],
-      [3, addReference],
-      [1, stopOrRestart],
-      [2, sleep]
-    ])
+    const op = freq(availableOps)
     ops.push(op)
   }
-  if (!running) {
-    ops.push({ op: 'restart' })
-  }
 }
 
-function generate () {
+function generateLocalWatcher () {
   let ops = []
   init(ops)
-  start(ops)
-  run(ops)
+  startWatcher(ops)
+  run(ops, localOps)
+  if (!running) {
+    ops.push({ op: 'restart' })
+  }
   console.log(JSON.stringify(ops))
 }
 
-generate()
+function generateTwoClients () {
+  let result = { desktop: [], laptop: [] }
```

Comment:
When variables are not reassigned, a lot of JS guidelines recommand using `const` instead of `let`: https://eslint.org/docs/rules/prefer-const